pos_order_zipcode/models/pos_config.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Moka Tourisme (https://www.mokatourisme.fr).
+# @author Pierre Verkest <pierreverkest84@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    pos_order_zipcode_zip_required = fields.Boolean(
```

Comment:
In a functionnal point of view, I see this part very bad. 
No personal data should be required to make a sale.
A customer can refuse to provide such information.
The result of such constraint is having database with incorrect data, because customer say anything (or cashier will enter anything).

Reply:
I'll add something in the road map something that cashier could click on a button to by pass that step anyway.

The purpose of this module is also to remind the cashier to ask about it, In the meantime I suppose they set a special code to exclude those orders.

As Today the analysis is done by an other software.